driver/lib/sessions/ios.ts

```diff
@@ -22,11 +22,25 @@ export const startIOSSession = async (caps) => {
   const observatoryWsUri = getObservatoryWsUri(iosdriver);
   return Promise.all([
     iosdriver,
-    connectSocket(observatoryWsUri, caps.retryBackoffTime, caps.maxRetryCount),
+    connectSocket(await observatoryWsUri, caps.retryBackoffTime, caps.maxRetryCount),
   ]);
 };
 
-export const getObservatoryWsUri = (proxydriver) => {
+export const getObservatoryWsUri = async (proxydriver) => {
```

Comment:
Perhaps await/async is not needed any more? https://github.com/truongsinh/appium-flutter-driver/pull/132